core/src/main/java/org/apache/calcite/runtime/SqlFunctions.java

```diff
@@ -3636,6 +3636,44 @@ public static Object[] array(Object... args) {
     return args;
   }
 
+  /**
+   * Returns whether there is an element in {@code list} for which {@code predicate} is true.
+   * Also, if {@code predicate} returns null for any element of {@code list}
+   * and there are no true comparison result is obtained, the result will be null, not false.
```

Comment:
This sentence should probably be reworked (i.e. "there are" and "is obtained").

Reply:
Fixed.